packages/contracts-bedrock/src/L2/SuperchainERC20.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.25;
 import { ICrosschainERC20 } from "src/L2/interfaces/ICrosschainERC20.sol";
 import { ISemver } from "src/universal/interfaces/ISemver.sol";
 import { Predeploys } from "src/libraries/Predeploys.sol";
-import { ERC20 } from "@solady/tokens/ERC20.sol";
+import { ERC20 } from "@solady-v0.0.245/tokens/ERC20.sol";
```

Comment:
Do we have any security justification for Solady's ERC20 implementation? Was it audited, and do we know any major projects that use it?

Reply:
Tracking issue: https://github.com/ethereum-optimism/optimism/issues/12573